ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/network/NoopP2PNetwork.java

```diff
@@ -61,12 +61,17 @@ public void subscribeDisconnect(final DisconnectCallback callback) {}
 
   @Override
   public boolean addMaintainedConnectionPeer(final Peer peer) {
-    throw new P2PDisabledException("P2P networking disabled.  Unable to connect to add peer.");
+    throw new P2PDisabledException("P2P networking disabled. Unable to connect to add peer.");
   }
 
   @Override
   public boolean removeMaintainedConnectionPeer(final Peer peer) {
-    throw new P2PDisabledException("P2P networking disabled.  Unable to remove a connected peer.");
+    throw new P2PDisabledException("P2P networking disabled. Unable to remove a connected peer.");
+  }
+
+  @Override
+  public Collection<Peer> getMaintainedConnectionPeers() {
+    throw new P2PDisabledException("P2P networking disabled. Unable to get peers.");
```

Comment:
We are a bit inconsistent in this class. In cases where we are getting peers (getPeers(), streamDiscoveredPeers(), and getMaintainedConnectionPeers()) we should either always throw an exception, or always return emtpty.